lib/galaxy/datatypes/converters/sam_to_bam.py

```diff
@@ -20,6 +21,17 @@ def cleanup_before_exit( tmp_dir ):
         shutil.rmtree( tmp_dir )
 
 
+def getSamtoolsVersion():
+    cmd = 'samtools --version'
```

Comment:
You could replace this function with https://gist.github.com/anonymous/409902c3cec78909873c8913b8f91135
I just adapted this slightly from https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/binary.py#L234